launcher-package/src/universal/bin/sbt.bat

```diff
@@ -526,25 +533,15 @@ goto args_loop
 rem Confirm a user's intent if the current directory does not look like an sbt
 rem top-level directory and the "new" command was not given.
 
-if not defined sbt_args_sbt_create if not defined sbt_args_print_version if not defined sbt_args_print_sbt_version if not defined sbt_args_print_sbt_script_version if not defined shutdownall if not exist build.sbt (
+if not defined sbt_args_allow_empty if not defined sbt_args_print_version if not defined sbt_args_print_sbt_version if not defined sbt_args_print_sbt_script_version if not defined shutdownall if not exist build.sbt (

```

Comment:
I think these extra `sbt_args_print_[version,sbt_version,sbt_script_version]` checks are still missing on the bash side?

Reply:
On Bash, I've refactored the check to `checkWorkingDirectory` function, and it's invoked right before sbt is about to actually call the launcher.

Reply:
Ok, missed that part.